### DISCUSSION TITLE
Sidekiq-ent default wait timeout on Limiters

### DISCUSSION THREAD

**Discussion by hype08** [sidekiq/sidekiq#5033] [Q&A] [answered]
In the wiki for [rate limiting](https://github.com/mperham/sidekiq/wiki/Ent-Rate-Limiting) it states Leaky limiters default to wait_timeout value of 5. 

Are other Limiter configurations also defaulted to the value of 5? I could not find anything in the wiki.

What if no wait_timeout argument is passed in to a Limiter? 

Many thanks!

**Comment by mperham** [comment]
`bundle open sidekiq-ent`?

**Comment by mperham** [comment, marked as answer]
Yes, `wait_timeout` defaults to 5 for all limiters.

**Reply by hype08** [reply]
Thank you!
